src/qt/walletcontroller.cpp

```diff
@@ -340,7 +349,11 @@ LoadWalletsActivity::LoadWalletsActivity(WalletController* wallet_controller, QW
 
 void LoadWalletsActivity::load()
 {
-    showProgressDialog(tr("Loading wallets…"));
+    showProgressDialog(
+      //: Title of progress window when which is displayed when wallets are being loaded.
+      tr("Load Wallet"),
+      //: Describing what is being done in background.
+      tr("Loading wallets…"));
```

Comment:
Please fix indentation.

nit: s/"Load Wallet"/"Load Wallets"/ (singular -> plural).

Reply:
@shaavan apply clang format:

```diff
@@ -350,10 +350,10 @@ LoadWalletsActivity::LoadWalletsActivity(WalletController* wallet_controller, QW
 void LoadWalletsActivity::load()
 {
     showProgressDialog(
-      //: Title of progress window when which is displayed when wallets are being loaded.
-      tr("Load Wallet"),
-      //: Describing what is being done in background.
-      tr("Loading wallets…"));
+        //: Title of progress window when which is displayed when wallets are being loaded.
+        tr("Load Wallet"),
+        //: Describing what is being done in background.
+        tr("Loading wallets…"));
 
     QTimer::singleShot(0, worker(), [this] {
         for (auto& wallet : node().walletClient().getWallets()) {
```

Reply:
> nit: s/"Load Wallet"/"Load Wallets"/ (singular -> plural).

Fixed

> apply clang-format:

Applied

Addressed in ccecfa8